README.md

```diff
@@ -63,6 +65,13 @@ GCC>=5 is ABI compatible for minor versions. To solve multiple minors, there are
 | - [conanio/clang7: clang 7](https://hub.docker.com/r/conanio/clang7/)                 | x86_64 |  Supported |
 
 
+#### Visual Studio
+
+| Version                                                                                       | Arch       |  Status, Life cycle  |
+|-----------------------------------------------------------------------------------------------|------------|------------|
+| - [conanio/msvc14: Visual Studio 2015 14.0](https://hub.docker.com/r/conanio/msvc14/) | x86_64/x86 |  Supported |
+| - [conanio/msvc15: Visual Studio 2017 15.0](https://hub.docker.com/r/conanio/msvc15/) | x86_64/x86 |  Supported |
```

Comment:
`14.0` and `15.0` aren't true AFAIK

It's more like the latest available patch version of 14 and 15, pretty much like we had it the entire time on AppVeyor. I would suggest to skip the minor version naming and just name the major.